vllm_ascend/envs.py

```diff
@@ -66,6 +66,15 @@
     lambda: os.getenv("C_COMPILER", None),
     "VLLM_VERSION":
     lambda: os.getenv("VLLM_VERSION", None),
+    # ACL_OP_INIT_MODE:
+    #   0: default, normal init.
+    #   1: delay init until launch aclops.
+    #   2: forbid aclops init and launch.
+    # Find more details at https://gitee.com/ascend/pytorch/pulls/18094
+    # We set this var default to `1` in vllm-ascend to avoid segment fault when
+    # enable `pin_memory` while creating a tensor using `torch.tensor`.
+    "ACL_OP_INIT_MODE":
+    lambda: os.getenv("ACL_OP_INIT_MODE", '1'),
```

Comment:
I just realize that add the env here doesn't work. Here is to get from env. But what you need is to set env.